CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Remove
+- Remove `anyhow` and `thiserror`. [#39](https://github.com/dusk-network/plonk_gadgets/issues/39)
```

Comment:
It looks to me that you also added and modified the tests: are all the addition / changes *only* related to the `anyhow` / `thiserror` removal, or there are tests that weren't present before that you added?

If that is the case, you should add in the changelog a section about those tests (and possible mention the same in the commit message)

Reply:
Are only related to anyhow.